en_US/messages.json

```
@@ -78,5 +78,35 @@
     "qualityEstimationMessage": {
       "message": "Enable quality estimation",
       "description":"Checkbox to enable the quality estimation of the translation."
+    },
+    "formtranslationsReady": {
+      "message": "Ready",
+      "description":"Notifies that the form translation is ready to be used."
+    },
+    "formtranslationsTyping": {
+      "message": "Typing",
+      "description":"Notifies the user that typing is occuring in the form translations area."
+    },
+    "formtranslationsTranslationInProgress": {
+      "message": "Translation in progress…",
+      "description":"Notifies the user that the translation of the input text is happening."
+    },
+    "formtranslationsTranslationCompleted": {
+      "message": "Translation completed.",
+      "description":"Notifies the user that the translation of the input text is completed."
+    },
+    "formtranslationsDescription": {
+      "message": "This is form translations by Firefox Translations. Type in the form to the left in <em>$navigatorLanguage$</em> and have the form you selected translated to <em>$pageLanguage$</em>. The form to the right represents the text translated back from <em>$pageLanguage$</em> to <em>$navigatorLanguage$</em> for verification.",
```

Thanks, the screencast helps a lot.

What do you think about this?

```suggestion
      "message": "This is the form translation feature by Firefox Translations. Type in the field on the left in <em>$navigatorLanguage$</em> and the translation in <em>$pageLanguage$</em> will be inserted in the original form. The field on the right shows the text translated back from <em>$pageLanguage$</em> to <em>$navigatorLanguage$</em> for verification purpose.",
```

---

This is great! Much better.